libcec/Makefile

```diff
@@ -43,4 +43,18 @@ define Package/libcec/install
 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/*.so* $(1)/usr/lib/
 endef
 
+
+define Package/cec-client
```

Comment:
Please group this section with Package/libcec, in alphabetical order.

Reply:
I'm just barely able to edit simple OpenWrt makefiles, and in no way an OpenWrt build system expert. Can you give me a hint what you mean with "grouping sections"? I found no hint in the wiki in [creating packages](https://openwrt.org/docs/guide-developer/packages) or [package policies](https://openwrt.org/docs/guide-developer/package-policies).

Reply:
```
define Package/cec-client
...
endef

define Package/libcec
...
endef

define Package/cec-client/install
...
endef

define Package/libcec/install
...
endef
```